wagtail/contrib/modeladmin/views.py

```diff
@@ -115,7 +115,7 @@ def get_edit_handler_class(self):
         if hasattr(self.model, 'edit_handler'):
             edit_handler = self.model.edit_handler
         else:
-            panels = extract_panel_definitions_from_model_class(self.model)
+            panels = extract_panel_definitions_from_model_class(self.model, exclude=self.model_admin.exclude_fields)
```

Comment:
The general pattern I've tried to follow with ModelAdmin is to accompany each attribute with a `get()` method, which just returns the attribute value by default. It would help developers when their needs go beyond defining a simple list. Do you think we might be able to do something similar here? Also, since the method would always be called with access to a request object, it might be handy for the method to accept `request` as a positional argument (which would give developers scope to exclude different fields depending on the user).